### ISSUE TITLE
IllegalArgumentException

### ISSUE THREAD

**Issue by 25** [25/EnchantmentsEnhance#94]
```
[12:15:43 INFO]: XiaoWeiXiao issued server command: /enhance menu
[12:15:43 WARN]: java.lang.IllegalArgumentException: Cannot set already-set enchantment
[12:15:43 WARN]:        at org.bukkit.enchantments.Enchantment.registerEnchantment(Enchantment.java:284)
[12:15:43 WARN]:        at org.pixeltime.enchantmentsenhance.compatibility.UnsafeGlow_V1_12$Constants$GLOW$2.invoke(UnsafeGlow_V1_12.kt:50)
[12:15:43 WARN]:        at org.pixeltime.enchantmentsenhance.compatibility.UnsafeGlow_V1_12$Constants$GLOW$2.invoke(UnsafeGlow_V1_12.kt:37)
[12:15:43 WARN]:        at kotlin.SynchronizedLazyImpl.getValue(LazyJVM.kt:74)
[12:15:43 WARN]:        at org.pixeltime.enchantmentsenhance.compatibility.UnsafeGlow_V1_12$Constants.getGLOW(UnsafeGlow_V1_12.kt)
[12:15:43 WARN]:        at org.pixeltime.enchantmentsenhance.compatibility.UnsafeGlow_V1_12$Constants.access$getGLOW$p(UnsafeGlow_V1_12.kt:37)
[12:15:43 WARN]:        at org.pixeltime.enchantmentsenhance.compatibility.UnsafeGlow_V1_12.addGlow(UnsafeGlow_V1_12.kt:33)
[12:15:43 WARN]:        at org.pixeltime.enchantmentsenhance.manager.modular.GlowItem_Unsafe.addGlow(GlowItem_Unsafe.java:37)
[12:15:43 WARN]:        at org.pixeltime.enchantmentsenhance.gui.menu.icons.StoreIcon.getGlowingItem(StoreIcon.java:40)
[12:15:43 WARN]:        at org.pixeltime.enchantmentsenhance.gui.menu.MainMenu.update(MainMenu.java:260)
[12:15:43 WARN]:        at org.pixeltime.enchantmentsenhance.gui.menu.MainMenu.<init>(MainMenu.java:82)
[12:15:43 WARN]:        at org.pixeltime.enchantmentsenhance.command.player.MenuCommand.onCommand(MenuCommand.kt:32)
[12:15:43 WARN]:        at org.pixeltime.enchantmentsenhance.manager.CommandManager.onCommand(CommandManager.java:135)
[12:15:43 WARN]:        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:44)
[12:15:43 WARN]:        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:141)
[12:15:43 WARN]:        at org.bukkit.craftbukkit.v1_12_R1.CraftServer.dispatchCommand(CraftServer.java:641)
[12:15:43 WARN]:        at net.minecraft.server.v1_12_R1.PlayerConnection.handleCommand(PlayerConnection.java:1385)
[12:15:43 WARN]:        at net.minecraft.server.v1_12_R1.PlayerConnection.a(PlayerConnection.java:1220)
[12:15:43 WARN]:        at net.minecraft.server.v1_12_R1.PacketPlayInChat.a(PacketPlayInChat.java:45)
[12:15:43 WARN]:        at net.minecraft.server.v1_12_R1.PacketPlayInChat.a(PacketPlayInChat.java:1)
[12:15:43 WARN]:        at net.minecraft.server.v1_12_R1.PlayerConnectionUtils$1.run(SourceFile:13)
[12:15:43 WARN]:        at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source)
[12:15:43 WARN]:        at java.util.concurrent.FutureTask.run(Unknown Source)
[12:15:43 WARN]:        at net.minecraft.server.v1_12_R1.SystemUtils.a(SourceFile:46)
[12:15:43 WARN]:        at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:747)
[12:15:43 WARN]:        at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:405)
[12:15:43 WARN]:        at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:678)
[12:15:43 WARN]:        at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:576)
[12:15:43 WARN]:        at java.lang.Thread.run(Unknown Source)
```

**Comment by 25** [comment]
Happened when servers are reloaded

**Comment by lgou2w** [comment]
Fixed #93
